File: scala/fizzbuzz.scala
(1 until 100).map(_ match {                
    case i: Int if ((i % 3) == 0 && (i % 5) == 0) => "FizzBuzz"
    case i: Int if ((i % 3) == 0) => "Fizz"
    case i: Int if ((i % 5) == 0) => "Buzz"
    case i: Int => i.toString
}).foreach(println _)

File: swift/fizzbuzz.swift
func fizzbuzz(until endValue:Int) {
    for i in 1...endValue {
        if i % 3 == 0 && i % 5 == 0 {
            print("FizzBuzz")
        } else {
            if i % 3 == 0 {
                print("Fizz")
            } else {
                if i % 5 == 0 {
                    print("Buzz")
                } else {
                    print(i)
                }
            }
        }
    }
}
